packages/page-claims/src/index.tsx

```diff
@@ -73,183 +68,242 @@ const Signature = styled.textarea`
+      {!isOldClaimProcess && <Warning />}
+      <h1>
+        <Trans>claim your <em>{TokenUnit.abbr}</em> tokens</Trans>
+      </h1>
+      <Columar>
+        <Column>
+          <Card withBottomMargin>
+            <h3>{t('1. Select your {{chain}} account', {
+              replace: {
+                chain: systemChain
+              }
+            })}</h3>
+            <InputAddress
+              defaultValue={accountId}
+              help={t('The account you want to claim to.')}
+              label={t('claim to account')}
+              onChange={setAccountId}
+              type='all'
+            />
+            {(step === Step.Account) && (
+              <Button.Group>
+                <Button
+                  icon='sign-in'
+                  isDisabled={preclaimEthereumAddress === PRECLAIMS_LOADING}
+                  label={preclaimEthereumAddress === PRECLAIMS_LOADING
+                    ? t('Loading')
+                    : t('Continue')
+                  }
+                  onClick={handleAccountStep}
+                />
+              </Button.Group>
+            )}
+          </Card>
+          {
+            // We need to know the ethereuem address only for the new process
+            // to be able to know the statement kind so that the users can sign it
+            (step >= Step.ETHAddress && !isPreclaimed && !isOldClaimProcess) && (
+              <Card withBottomMargin>
+                <h3>{t('2. Enter the ETH address from the sale.')}</h3>
+                <Input
+                  autoFocus
+                  className='full'
+                  help={t('The the Ethereum address you used during the pre-sale (starting by "0x")')}
+                  label={t('Pre-sale ethereum address')}
+                  onChange={onChangeEthereumAddress}
+                  value={ethereumAddress || ''}
+                />
+                {(step === Step.ETHAddress) && (
+                  <Button.Group>
+                    <Button
+                      icon='sign-in'
+                      isDisabled={!ethereumAddress}
+                      label={t('Continue')}
+                      onClick={goToStepSign}
+                    />
+                  </Button.Group>
+                )}
+              </Card>
+            )}
+          {(step >= Step.Sign && !isPreclaimed) && (
+            <Card>
+              {(isOldClaimProcess || !!statementKind)
+                ? <>
+                  <h3>{t('{{step}}. Sign with you ETH address',
+                    { replace: { step: isOldClaimProcess ? '2' : '3' } })}</h3>
+                  {!isOldClaimProcess && (<Statement
+                    kind={statementKind}
+                  />) }
+                  <div>
+                    {t('Copy the following string and sign it with the Ethreum account you used during the pre-sale in the wallet of your choice, using the string as the payload, and then paste the transaction signature object below')}
+                    :
```

Comment:
No sure that this `:` is here - is it part of the text, the unreadable `?:` or something else?

Reply:
Use `<div>{t('...')}</div>` since it is a single element